.github/workflows/python-build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build Python Package
```

Comment:
IIUC, this new workflow is running in parallel with all other workflows, so this will build python packages even when our tests are failing. Is that ok?

Unless I'm missing something, I would prefer to build this only when tests are passing.

Also, about the presentation of multiple pipelines, I would follow the same comment that I wrote for ana here:

https://github.com/avocado-framework/avocado/pull/4698#pullrequestreview-686373273

Reply:
> IIUC, this new workflow is running in parallel with all other workflows, so this will build python packages even when our tests are failing. Is that ok?
> 

I think it's intended.  This is indeed only building the packages, so it effectively tests the build.  Now, let's say that something is introduced in `setup.py` that doesn't impact `develop` or tests, but would still break the package generation: these would catch that.  You can look at the `MANIFEST.in` file change, and how what I describe here is a valid scenario.

> Unless I'm missing something, I would prefer to build this only when tests are passing.
> 

I disagree, based on the example above.  I see no reason for not making these run in parallel.

> Also, about the presentation of multiple pipelines, I would follow the same comment that I wrote for ana here:
> 
> [#4698 (review)](https://github.com/avocado-framework/avocado/pull/4698#pullrequestreview-686373273)

Do you mean moving these to `ci.yml`?